test/dynamo/test_ctx_manager.py

```diff
@@ -1838,14 +1846,18 @@ def fn(x):
 
 class ContextlibContextManagerTests(torch._dynamo.test_case.TestCase):
     def setUp(self):
+        super().setUp()
         self._prev = torch._dynamo.config.enable_trace_contextlib
         self._u_prev = torch._dynamo.config.enable_trace_unittest
         torch._dynamo.config.enable_trace_contextlib = True
         torch._dynamo.config.enable_trace_unittest = True
+        torch._dynamo.config.nested_graph_breaks = True
 
     def tearDown(self):
+        super().tearDown()
         torch._dynamo.config.enable_trace_contextlib = self._prev
         torch._dynamo.config.enable_trace_unittest = self._u_prev
+        torch._dynamo.config.nested_graph_breaks = False
```

Comment:
Shouldn't you save the previous value? Otherwise, you may falsify a flag that was originally `True`.